script/test_external

```diff
@@ -1,9 +1,3 @@
 #!/bin/bash
 
-SCRIPT_DIR=$(dirname "$(readlink -f $0)")
-. ${SCRIPT_DIR}/lib/functions
-
-start_services
-bundle_exec 3scale_backend start -p 3000 $@
-stop_services
-
+bundle exec 3scale_backend start -p 3000 $@
```

Comment:
Hm, it seems to me that this script doesn't have much value anymore... It used to start the required services, but it doesn't anymore, so maybe it can go? 
We have some instructions on how to start apisonator locally, don't we? (a genuine question, I haven't read through all the updated docs yet)

Reply:
Yeah, not really needed anymore. Docs mention this script, that's why I kept it, but it  only run one command so better to directly mention the command in the docs and get rid of more junk.